.github/workflows/cd.yml

```diff
@@ -27,104 +27,104 @@ jobs:
-        uses: actions/download-artifact@v1
-        with:
-          name: dockerfile-image-update-itest
-      - name: integration test
-        env:
-          ITEST_GH_TOKEN: ${{ secrets.ITEST_GH_TOKEN }}
-        run: echo "Testing new version ${{ needs.get-next-version.outputs.version }}" && make integration-test
-  codecov:
-    runs-on: ubuntu-latest
-    needs: build-test
-    steps:
-      - name: Download main project dir
-        uses: actions/download-artifact@v1
-        with:
-          name: dockerfile-image-update
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v1.0.10
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          file: ./dockerfile-image-update/target/site/jacoco/jacoco.xml
-          # flags: unittests
-          # yml: ./codecov.yml
-          fail_ci_if_error: true
-  codeclimate:
-    runs-on: ubuntu-latest
-    needs: build-test
-    steps:
-      - name: checkout
-        uses: actions/checkout@master
-        with:
-          ref: ${{ github.ref }}
-      - name: Download main project dir
-        uses: actions/download-artifact@v1
-        with:
-          name: dockerfile-image-update
-      - name: Upload coverage to Code Climate
-        uses: paambaati/codeclimate-action@v2.6.0
-        env:
-          CC_TEST_REPORTER_ID: ${{ secrets.CODE_CLIMATE_REPORTER_ID }}
-          JACOCO_SOURCE_PATH: "${{github.workspace}}/dockerfile-image-update/src/main/java"
-        with:
-          # The report file must be there, otherwise Code Climate won't find it
-          coverageCommand: echo "already done"
-          coverageLocations: |
-            ${{github.workspace}}/dockerfile-image-update/target/site/jacoco/jacoco.xml:jacoco
-  master-deploy:
-    needs: [integration-test, get-next-version]
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/master'
-    steps:
-      - name: checkout
-        uses: actions/checkout@master
-        with:
-          ref: ${{ github.ref }}
-      - name: Download main project dir
-        uses: actions/download-artifact@v1
-        with:
-          name: dockerfile-image-update
-      - name: deploy
-        env:
-          encrypted_00fae8efff8c_iv: ${{ secrets.encrypted_00fae8efff8c_iv }}
-          encrypted_00fae8efff8c_key: ${{ secrets.encrypted_00fae8efff8c_key }}
-          encrypted_96e73e3cb232_iv: ${{ secrets.encrypted_96e73e3cb232_iv }}
-          encrypted_96e73e3cb232_key: ${{ secrets.encrypted_96e73e3cb232_key }}
-          GPG_KEY_NAME: ${{ secrets.GPG_KEY_NAME }}
-          GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
-          DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
-          DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
-          CI_DEPLOY_USER: ${{ secrets.CI_DEPLOY_USER }}
-          CI_DEPLOY_PASSWORD: ${{ secrets.CI_DEPLOY_PASSWORD }}
-          NEW_PATCH_VERSION: ${{ needs.get-next-version.outputs.version }}
-        run: make deploy
+      # - name: Upload itest module
```

Comment:
Could you help me understand why you're doing this? Ideally, you shouldn't need to modify our CI.

Reply:
Hi, this was just for our local, please ignore changes to CI/CD.